src/styles/01-atoms/text/text/_text.scss

```diff
@@ -75,6 +74,10 @@ p,
     list-style: initial;
     color: $gray-medium;
   }
+
+  &__list-item {
+    list-style-position: inside;
```

Comment:
Do the designs not hang their punctuation? If so we should talk with Ellen about it, it makes content jump horizontally a lot if not

Reply:
I ... am not sure what this means?  I was trying to solve this issue where it looks different in mobile and desktop widths:  

mobile: 
![Screen Shot 2020-04-20 at 11 26 14 AM](https://user-images.githubusercontent.com/3579472/79769200-cf068500-82f9-11ea-94cb-e338d357ff68.png)

desktop
![Screen Shot 2020-04-20 at 11 26 23 AM](https://user-images.githubusercontent.com/3579472/79769206-d0d04880-82f9-11ea-93ec-53dcfe1991be.png)

Reply:
Ah, I see. I would request that you push back on Ellen for these designs—it's odd that they're being hidden on mobile, and I don't think that unordered lists should hide their bullets in mobile for the DS at least. If it's necessary that they do that should only exist on SFR.

Reply:
sorry, I think I wasn't clear - without the change, this is what it looks like.  You can check it out here:  http://researchnow-beta.nypl.org/
With this change, the bullets will no longer hide in mobile, which is what we want.  But if there's a better way to make the bullets not hide in mobile, I'm all up for it!

Reply:
Ah, I see what you're saying. I have some thoughts on ways to do this but not anything that fixes it quickly. I'll resolve this, carry on.